cli/azd/pkg/commands/pipeline/azdo_provider.go

```diff
@@ -596,8 +596,16 @@ func (p *AzdoCiProvider) configureConnection(
 	repoDetails *gitRepositoryDetails,
 	provisioningProvider provisioning.Options,
 	credentials json.RawMessage,
+	authType PipelineAuthType,
 	console input.Console) error {
 
+	if authType == AuthTypeOidc {
+		return fmt.Errorf(
+			"Azure DevOps does not support OIDC authentication. Service Principal with client ID and client secret must be used. %w",
+			ErrAuthNotSupported,
+		)
```

Comment:
Wondering if we could just `ignore the auth type` for azdo and show a note: "auth types not supported for Azdo. Using secret client..."

Reply:
This error would only show if they explicitly tried to set `--auth-type` flag to `federated`.  Otherwise it will just continue like normal.